includes/admin/class-wp-job-manager-settings.php

```diff
@@ -455,24 +455,36 @@ protected function init_settings() {
 						],
 					],
 				],
-				'recaptcha'      => [
-					__( 'reCAPTCHA', 'wp-job-manager' ),
+				'captcha'        => [
+					__( 'CAPTCHA', 'wp-job-manager' ),
 					[
 						[
 							'name'        => 'job_manager_recaptcha_label',
 							'std'         => __( 'Are you human?', 'wp-job-manager' ),
 							'placeholder' => '',
 							'label'       => __( 'Field Label', 'wp-job-manager' ),
-							'desc'        => __( 'The label used for the reCAPTCHA field on forms.', 'wp-job-manager' ),
+							'desc'        => __( 'The label used for the CAPTCHA field on forms.', 'wp-job-manager' ),
 							'attributes'  => [],
 						],
+						[
+							'name'        => 'job_manager_recaptcha_version',
+							'std'         => '',
+							'placeholder' => '',
+							'label'       => __( 'reCAPTCHA Version', 'wp-job-manager' ),
+							'desc'        => 'Choose between reCAPTCHA v2 or v3. Note: you will need API keys for the specific version you choose.',
```

Comment:
We probably want to translate this, right?

```suggestion
							'desc'        => __( 'Choose between reCAPTCHA v2 or v3. Note: you will need API keys for the specific version you choose.', 'wp-job-manager' ),
```

Reply:
Done in 475b4b11a90d40396322b0820afab0253d6d2995